AppPackage/Package.swift

```diff
@@ -0,0 +1,36 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "AppPackage",
+    platforms: [
+        .iOS(.v14),
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "AppPackage",
+            targets: ["AppPackage"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/nalexn/ViewInspector", from: "0.9.3"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "AppPackage",
+            dependencies: [
+            ]),
+        .testTarget(
+            name: "AppPackageTests",
+            dependencies: [
+                "AppPackage",
+                "ViewInspector",
+            ]),
```

Comment:
<!--
  0 failure: 
  1 warning:  Multiline argumen...
  
  
  DangerID: danger-id-Danger;
  File: AppPackage/Package.swift;
  Line: 34;
-->

- :warning: Multiline arguments should have their surrounding brackets in a new line. (`multiline_arguments_brackets`)